v2/bigquery-to-bigtable/src/main/java/com/google/cloud/teleport/v2/templates/BigQueryToBigtable.java

```diff
@@ -118,6 +119,60 @@ public interface BigQueryToBigtableOptions extends PipelineOptions {
     String getBigtableWriteColumnFamily();
 
     void setBigtableWriteColumnFamily(String value);
+
+    @TemplateParameter.Text(
+        order = 9,
+        optional = true,
+        description = "Bigtable's latency target in milliseconds for latency-based throttling",
+        helpText = "This enables latency-based throttling and specifies the target latency")
+    String getBigtableBulkWriteLatencyTargetMs();
+
+    void setBigtableBulkWriteLatencyTargetMs(String value);
+
+    @TemplateParameter.Text(
+        order = 10,
+        optional = true,
+        description = "The max number of row keys in a Bigtable batch write operation",
+        helpText = "This sets the max number of row keys in a Bigtable batch write operation")
+    String getBigtableBulkWriteMaxRowKeyCount();
+
+    void setBigtableBulkWriteMaxRowKeyCount(String value);
+
+    @TemplateParameter.Text(
+        order = 11,
+        optional = true,
+        description = "The max amount of bytes in a Bigtable batch write operation",
+        helpText = "This sets the max amount of bytes in a Bigtable batch write operation")
+    String getBigtableBulkWriteMaxRequestSizeBytes();
+
+    void setBigtableBulkWriteMaxRequestSizeBytes(String value);
+
+    @TemplateParameter.Text(
+        order = 12,
+        optional = true,
+        description = "The timeout for an RPC attempt in milliseconds",
+        helpText = "This sets the timeout for an RPC attempt in milliseconds")
+    String getBigtableRpcAttemptTimeoutMs();
+
+    void setBigtableRpcAttemptTimeoutMs(String value);
+
+    @TemplateParameter.Text(
+        order = 13,
+        optional = true,
+        description = "The total timeout for an RPC operation in milliseconds",
+        helpText = "This sets the total timeout for an RPC operation in milliseconds")
+    String getBigtableRpcTimeoutMs();
+
+    void setBigtableRpcTimeoutMs(String value);
+
+    @TemplateParameter.Text(
+        order = 14,
+        optional = true,
+        description = "The additional retry codes",
+        helpText = "This sets the additional retry codes")
+    String getBigtableAdditionalRetryCodes();
```

Comment:
Have an example of how this looks like? `401,419`? Please add here. The `@TemplateParameter` annotations accept `example = `

Reply:
Done.